patches/rrweb@2.0.0-alpha.13.patch

```diff
@@ -1,812 +0,0 @@
-+                    break;
-+                }
-+                case IncrementalSource.AdoptedStyleSheet: {
-+                    if (this.usingVirtualDom)
-+                        this.adoptedStyleSheets.push(d);
-+                    else
-+                        this.applyAdoptedStyleSheet(d);
-                     break;
-                 }
--                this.applySelection(d);
--                break;
--            }
--            case IncrementalSource.AdoptedStyleSheet: {
--                if (this.usingVirtualDom)
--                    this.adoptedStyleSheets.push(d);
--                else
--                    this.applyAdoptedStyleSheet(d);
--                break;
-             }
-+        } catch (error) {
-+            this.config.onError(error);
-         }
-     }
-     applyMutation(d, isSync) {
-diff --git a/es/rrweb/packages/rrweb/src/replay/media/index.js b/es/rrweb/packages/rrweb/src/replay/media/index.js
-index 22fee601e786c1d8dfb5c01d2e359c8bcbac7c42..6d80d33030c601fa74f1b95573cb1590c3a0116b 100644
---- a/es/rrweb/packages/rrweb/src/replay/media/index.js
-+++ b/es/rrweb/packages/rrweb/src/replay/media/index.js
-@@ -21,7 +21,7 @@ class MediaManager {
-         this.mediaMap.forEach((mediaState, target) => {
-             this.syncTargetWithState(target);
-             if (options.pause) {
--                target.pause();
-+                target?.pause();
-             }
-         });
-     }
-@@ -49,7 +49,7 @@ class MediaManager {
-             target.currentTime = seekToTime;
-         }
-         else {
--            target.pause();
-+            target?.pause();
-             target.currentTime = mediaState.currentTimeAtLastInteraction;
-         }
-     }
-@@ -117,7 +117,7 @@ class MediaManager {
-                 void target.play();
-             }
-             else {
--                target.pause();
-+                target?.pause();
-             }
-         }
-         catch (error) {
-@@ -141,7 +141,7 @@ class MediaManager {
-             isPlaying = target.getAttribute('autoplay') !== null;
-         }
-         if (isPlaying && playerIsPaused)
--            target.pause();
-+            target?.pause();
-         let playbackRate = 1;
-         if (typeof mediaAttributes.rr_mediaPlaybackRate === 'number') {
-             playbackRate = mediaAttributes.rr_mediaPlaybackRate;
-@@ -180,6 +180,8 @@ class MediaManager {
-         this.syncTargetWithState(target);
-     }
-     mediaMutation({ target, timeOffset, mutation, }) {
-+        if (!['AUDIO', 'VIDEO'].includes(target.nodeName))
-+            return;
-         this.mediaMap.set(target, this.getMediaStateFromMutation({
-             target,
-             timeOffset,
-diff --git a/es/rrweb/packages/rrweb-snapshot/es/css.js b/es/rrweb/packages/rrweb-snapshot/es/css.js
-new file mode 100644
-index 0000000000000000000000000000000000000000..0a6c5930c017852afd3d7f0170f6eca821619dd3
---- /dev/null
-+++ b/es/rrweb/packages/rrweb-snapshot/es/css.js
```

Comment:
Now that https://github.com/rrweb-io/rrweb/pull/1458 we don't need any of the custom CSS stuff in a separate patch (basically everything below this line until the end of the file)

Reply:
that's very exciting